### PR TITLE
New package: KuroGames.PunishingGrayRaven version 1.0

### PR DESCRIPTION
**Pull request by solomoncyj**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/154716)

### CODE REVIEW

**Comment by Trenly** [comment]
This application doesn't appear to write any ARP entries; Is it a portable application?
```
Found KRInstall PGR Overseas [KuroGames.PunishingGrayRaven] Version 1.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://zspms-volcdn-gamestarter.kurogame.net/install/pclauncher.exe
  ██████████████████████████████   126 MB /  126 MB
Successfully verified installer hash
Starting package install...
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs>
```

File: manifests/k/KuroGames/PunishingGrayRaven/1.0/KuroGames.PunishingGrayRaven.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: KuroGames.PunishingGrayRaven
+PackageVersion: '1.0'
+PackageLocale: en-US
+Publisher: 广州库洛科技有限公司
```
Comment (Trenly) [inline manifests/k/KuroGames/PunishingGrayRaven/1.0/KuroGames.PunishingGrayRaven.locale.en-US.yaml:7]:
```suggestion
Publisher: KURO TECHNOLOGY (HONG KONG) CO., LTD.
```
This should match control panel

File: manifests/k/KuroGames/PunishingGrayRaven/1.0/KuroGames.PunishingGrayRaven.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: KuroGames.PunishingGrayRaven
+PackageVersion: '1.0'
+PackageLocale: en-US
+Publisher: 广州库洛科技有限公司
+PackageName: KRInstall PGR Overseas
```
Comment (Trenly) [inline manifests/k/KuroGames/PunishingGrayRaven/1.0/KuroGames.PunishingGrayRaven.locale.en-US.yaml:8]:
```suggestion
PackageName: Punishing Gray Raven
```
This should match control panel

File: manifests/k/KuroGames/PunishingGrayRaven/1.0/KuroGames.PunishingGrayRaven.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: KuroGames.PunishingGrayRaven
+PackageVersion: '1.0'
```
Comment (Trenly) [inline manifests/k/KuroGames/PunishingGrayRaven/1.0/KuroGames.PunishingGrayRaven.installer.yaml:5]:
```suggestion
PackageVersion: '1.4.2.0'
```
This should match control panel

File: manifests/k/KuroGames/PunishingGrayRaven/1.0/KuroGames.PunishingGrayRaven.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: KuroGames.PunishingGrayRaven
+PackageVersion: '1.0'
```
Comment (Trenly) [inline manifests/k/KuroGames/PunishingGrayRaven/1.0/KuroGames.PunishingGrayRaven.locale.en-US.yaml:5]:
```suggestion
PackageVersion: '1.4.2.0'
```
This should match control panel

File: manifests/k/KuroGames/PunishingGrayRaven/1.0/KuroGames.PunishingGrayRaven.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: KuroGames.PunishingGrayRaven
+PackageVersion: '1.0'
+InstallerType: nullsoft
```
Comment (Trenly) [inline manifests/k/KuroGames/PunishingGrayRaven/1.0/KuroGames.PunishingGrayRaven.installer.yaml:6]: Although it says it installs successfully, in a clean VM, the install doesn't succeed. Perhaps there are different installer switches that are needed?

File: manifests/k/KuroGames/PunishingGrayRaven/1.0/KuroGames.PunishingGrayRaven.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: KuroGames.PunishingGrayRaven
+PackageVersion: '1.0'
```
Comment (Trenly) [inline manifests/k/KuroGames/PunishingGrayRaven/1.0/KuroGames.PunishingGrayRaven.yaml:5]:
```suggestion
PackageVersion: '1.4.2.0'
```
This should match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]
Downloaded and ran the installer manually - seems the application does write ARP entries. This means something in the manifest is incorrect - either the installer type or the silent switches, perhaps a missing dependency, not sure what it is

**Comment by solomoncyj** [comment]
.

**Comment by stephengillie** [comment]
This package doesn't appear to install. These are the only files changed in the system during the install window:

image

**Pull request closed without merging**